package.json

```diff
@@ -46,7 +46,6 @@
     "array-last": "^1.1.1",
     "babylon": "^6.9.1",
     "compose-function": "^3.0.3",
-    "deep-freeze": "0.0.1",
```

Comment:
The practical supply chain security risk is actually pretty low since this dependency is pinned to an exact version. That is, even if a malicious actor takes over the account that owns `deep-freeze` and publishes a new version that is malicious, neither this project nor its users are affected because it won't be upgraded to automatically.